CONTRIBUTING.md

```diff
@@ -83,11 +83,14 @@ Code from treating its files as project files.
 
 #### Run a specific test
 
-To run a single test in VSCode, run the `Extension Tests (current file)` launch-config.
-
-Alternatively:
+To run a single test in VSCode, do any one of:
 
+-   Run the _Extension Tests (current file)_ launch-config.
 -   Use Mocha's [it.only()](https://mochajs.org/#exclusive-tests) or `describe.only()`.
+-   Run in your terminal:
+    ```
+    NO_COVERAGE=true TEST_FILE=src/test/foo.test npm run test
```

Comment:
Is this a Mac/Linux only instruction, or does this work on Windows as well?

Reply:
Looks like this works in Powershell: `$Env:NO_COVERAGE = "true"; $Env:TEST_FILE = ".\src\test\cloudWatchLogs\commands\addLogEvents.test.ts"; npm run test`

The above works fine in MinGW bash.

Reply:
Main point with this kind of doc is to give users the "secret" information. Not to list syntax variants (which is not secret).

Reply:
We've had instructions in the repo in the past that caused confusion for people who tried to pull down and work with the repo. In a contributing doc (whose target audience is developers), why wouldn't we provide turn-key instructions that assist in the development process for all supported systems?

Reply:
Depends on how verbose we are willing to make the docs.

Reply:
Can you check if this works in Windows (specifically with the `/` slashes and the lack of the `.ts` extension):

        $Env:NO_COVERAGE = "true"; $Env:TEST_FILE = "src/test/cloudWatchLogs/commands/addLogEvents.test"; npm run test

Reply:
Confirmed working. Strangely, ran the same query with `.ts` appended and it didn't actually pick up files; the one you posted is more correct.

Reply:
`TEST_FILE` wants the module name, i.e. without `.ts`. We should probably accept either in testRunner though.